jaxdecomp/_src/fft.py

```diff
@@ -61,10 +62,15 @@ def sfft(x,
   (x,) = promote_dtypes_complex(x)
 
   return sfft_p.bind(
-      x, fft_type=typ, pdims=pdims, global_shape=global_shape, adjoint=adjoint)
+      x,
+      fft_type=typ,
+      pdims=pdims,
+      global_shape=global_shape,
+      adjoint=adjoint,
+      yz_slab=yz_slab)
```

Comment:
Here `yz_slab` last after `adjoint`

Reply:
Same, YZ_slab is no longer used in last commit